modules/SofaTopologyMapping/Quad2TriangleTopologicalMapping.cpp

```diff
@@ -267,14 +272,14 @@ void Quad2TriangleTopologicalMapping::updateTopologicalMappingTopDown()
         }
         case core::topology::QUADSREMOVED:
         {
-            const sofa::helper::vector<unsigned int> &tab = ( static_cast< const QuadsRemoved *>( *itBegin ) )->getArray();
+            const auto & tab = ( static_cast< const QuadsRemoved *>( *itBegin ) )->getArray();
```

Comment:
I have nothing against using auto type, but in those cases maybe changing the name of the variable would be good.
Because ```auto tab ``` gives no info of what we are manipulating.  maybe tabIndices or what you think is best.

Reply:
I went to the file very quickly changing to auto where my IDE would suggest to do so (and a bit more), I agree that the name isn't very well chosen. I will try if I have some time to change the names... but here the idea of auto is more that it will automatically handle the case where the return value type changes in the future.